template-troubleshooting.md

```diff
@@ -0,0 +1,22 @@
+# Troubleshooting template
+
+When writing guidance to help to troubleshoot specific errors, the error must include:
+
```

Comment:
Maybe also have guidance on the Error Description here, ie how long it should be, what it should include, what it should not include etc.